src/main/java/com/app/service/graph/GraphService.java

```diff
@@ -1,32 +1,140 @@
 package com.app.service.graph;
 
+import javafx.fxml.FXML;
+import javafx.scene.Parent;
 import javafx.scene.layout.AnchorPane;
+import javafx.stage.FileChooser;
 import javafx.stage.Stage;
+import org.jfree.chart.JFreeChart;
 import org.jfree.chart.fx.ChartViewer;
 
+import java.awt.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+
 
 public class GraphService {
-    AnchorPane anchorPane;
-    Stage stage;
```

Comment:
Pamatas si clean code practices? :) vsetko co sa da schovat, schovaj pred vonkajskom, strel tam private aby to bolo clear clear, a nie len attributes ale aj functions, store vonku nepouzivas